[microstack] Initial build of content

pmahnke opened the pull request:
## Done

- Built microstack hompage based on the [design](https://github.com/canonical-web-and-design/web-squad/issues/1623) and [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- Added a single page of docs
- Added new GTM/GA codes

## QA

- Download this branch
- Go to http://0.0.0.0:8039/
- See that the copy matches the [copy doc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- See that the design matches the [design](https://github.com/canonical-web-and-design/web-squad/issues/1623)
- Gi ti http://0.0.0.0:8039/docs
- See that the docs look ok and largely are based on [this site](https://opendev.org/x/microstack/src/branch/master/README.md)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1624

## Screenshot

![image](https://user-images.githubusercontent.com/441217/64843450-10fd7900-d5fd-11e9-9fdd-8eed5a170b34.png)

webteam-app commented:
Starting demo at: https://microstack-run-canonical-web-and-design-pr-2.run.demo.haus/

jpmartinspt commented:
Fix comments

anasereijo commented:
- [ ] could we update the logos to match what we have on microk8s?

<img width="1172" alt="Screenshot 2019-09-13 at 10 17 41" src="https://user-images.githubusercontent.com/36884067/64851794-0ac4c800-d610-11e9-9e67-da62f44eeff1.png">

- [ ] should we have the git buttons here too?

- [ ] can we make `class="p-list__item is-ticked"` ubuntu orange?

- [ ] instead of the full-width grey separator lines could we have the `hr` ones? as per my design

![MicroStack - OpenStack in a snap](https://user-images.githubusercontent.com/36884067/64852126-af470a00-d610-11e9-89ce-a5a51537ac2b.png)

- [ ] on mobile could you fix the list spacing?

<img width="645" alt="Screenshot 2019-09-13 at 10 25 53" src="https://user-images.githubusercontent.com/36884067/64852270-f2a17880-d610-11e9-9d58-9c4fc94f11e8.png">

- [ ] hide the following line on mobile

<img width="565" alt="Screenshot 2019-09-13 at 10 26 53" src="https://user-images.githubusercontent.com/36884067/64852334-16fd5500-d611-11e9-8b72-2281005b296f.png">

pmahnke commented:
@anasereijo

1. the project isn't on github, so no buttons
2. the lists gap - I fixed it, but perhaps we should fix vanilla
3. the reliable, ready to go line on mobile is part of the matrix pattern, I fixed it, but we should perhaps add it to vanilla
